src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -91,7 +94,17 @@ public AttachCaseCallbackService(CcdApi ccdApi) {
             .map(attachCaseResult ->
                 attachCaseResult.map(modifiedFields -> mergeCaseFields(exceptionRecord.getData(), modifiedFields))
             )
+            .peek(result -> processPayment(result, exceptionRecord))
             .getOrElseGet(errors -> Either.left(errors.toJavaList()));
+
+    }
+
+    private void processPayment(Either<List<String>, Map<String, Object>> result, CaseDetails exceptionRecord) {
+        if (result.isRight() && result.get() != null) {
+            Long caseId = Long.valueOf((String) result.get().get(ATTACH_TO_CASE_REFERENCE));
+            log.info("Call update payment processor, caseId {}", caseId);
```

Comment:
there's a log already present in processor `Sending payment update message. ER id: {}`. We are overusing logs :( - we can just extend that one